tracer/src/Datadog.Trace/Iast/Aspects/DebugAspects.cs

```diff
@@ -25,6 +31,18 @@ public static string AspectMethodReplace(string target, string param1)
         return string.Concat(target, param1);
     }
 
+    /// <summary>
+    /// AspectMethodReplace test method
+    /// </summary>
+    /// <param name="target">main object instance </param>
+    /// <returns>target concatenated to param1</returns>
+    public static string AspectMethodReplace(object target)
+    {
+        Console.WriteLine($"[AspectMethodReplace]DebugAspects.AspectMethodReplace(object {target})");
```

Comment:
It's for testing purpose only (but needed as today's tests)

Reply:
This is meant to be used only on the `CallTargetNativeTest` app. This will never be executed on a client. I know it's not the best solution, but it's how we've done it so far. I'm open to improvement suggestion, but should be for another PR.